Formula/minicom.rb

```diff
@@ -20,15 +21,26 @@ class Minicom < Formula
     sha256 x86_64_linux:   "7d1b0aae1f169968d42e4dea644dff5a4f18010b59b334439aa2bd276c6e913a"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+
+  uses_from_macos "libiconv"
   uses_from_macos "ncurses"
 
   def install
     # There is a silly bug in the Makefile where it forgets to link to iconv. Workaround below.
     ENV["LIBS"] = "-liconv" if OS.mac?
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--mandir=#{man}"
+    if build.head?
+      system "./autogen.sh"
+      system "./configure", *std_configure_args, "--prefix=#{prefix}", "--mandir=#{man}"
+    else
+      system "./configure", "--disable-dependency-tracking",
+                            "--prefix=#{prefix}",
+                            "--mandir=#{man}"
+    end
```

Comment:
Why not
```suggestion
    system "./autogen.sh" if build.head?
    system "./configure", *std_configure_args, "--prefix=#{prefix}", "--mandir=#{man}"
```
?

Reply:
Right, I wasn't so sure about the `configure` arguments so I decided to leave the original version.
But AFAIK it should be fine, even `./autogen.sh` for the stable build (according to the README)